### ISSUE TITLE
Only display icons for navigation group of menus

### ISSUE THREAD

**Issue by rebornix** [Microsoft/vscode#27654]
re https://github.com/Microsoft/vscode/issues/27467

For commands in `navigation` group, right now we'll try to show icons first if they exist. If not, we display the command title. Our builtin explorer title only shows icons, so maybe we should only allow icons to be displayed as well. Allowing text being displayed sometimes looks weird.

![screen shot 2017-05-30 at 3 10 55 pm](https://cloud.githubusercontent.com/assets/876920/26607205/32ceb3fa-454a-11e7-8550-440f4142dafd.png)

**Comment by sandy081** [comment]
Its a good point. These actions are contributable. Have to see how they are rendered in other locations.

**Comment by bpasero** [comment]
The SCM viewlet has a similar "issue" where I would like to see an icon and not the text:

![image](https://cloud.githubusercontent.com/assets/900690/26636331/b755c3f2-461c-11e7-9e68-95dacdb7e19d.png)

I think today we are not restricting any extension to force use an icon, e.g. labels are allowed. @jrieken would know for sure if that is true.

**Comment by jrieken** [comment]
Yes, not restrictions and I'd that's fair. Tho it should not look ugly if no icons are provided, e.g no spacing issues like above.
